Description: Use system JFlex in /usr/share/java/JFlex.jar
Author: Damien Raude-Morvan <drazzib@debian.org>
Last-Update: 2009-11-13
Forwarded: not-needed
--- a/build/build.xml
+++ b/build/build.xml
@@ -39,7 +39,7 @@
       <arg value="--jlex" />
       <arg value="--nobak" />
       <arg value="${jflexfile}" />
-      <classpath path="JFlex.jar" />
+      <classpath path="/usr/share/java/JFlex.jar" />
     </java>
     <copy file="Yylex.java" tofile="${genlextargetfile}" />
   </target>
